server/index.js

```diff
@@ -44,6 +44,7 @@ app.get(`/api/v${__VERSION}/instructor/dashboard`, (req, res, next) => {
     }
 
     let token = header_validation.token;
+
```

Comment:
I think this line has some redundant whitespaces.

Reply:
Correct, will fix.